src/System.Reflection/tests/CoreCLR/Assembly/AssemblyTests.CoreCLR.cs

```diff
@@ -35,40 +32,34 @@ public void CurrentAssemblyHasLocation()
         }
 
         [Fact]
-        public void LoadFromStreamHasEmptyLocation()
+        public void LoadFromStream_Location_IsEmpty()
         {
             Assembly assembly = new TestStreamLoadContext().LoadFromAssemblyName(new AssemblyName("TinyAssembly"));
-            string location = assembly.Location;
-
-            Assert.NotNull(location);
-            Assert.Empty(location);
+            Assert.Empty(assembly.Location);
         }
 
         [Fact]
-        public void DynamicAssemblyThrowsNotSupportedException()
+        public void DynamicAssembly_Location_ThrowsNotSupportedException()
         {
             AssemblyBuilder builder = AssemblyBuilder.DefineDynamicAssembly(new AssemblyName("dynamic"), AssemblyBuilderAccess.Run);
-            Assert.Throws<NotSupportedException>(() => builder.Location); // wish this were Assert.Empty, but it's long been this way...
+            Assert.Throws<NotSupportedException>(() => builder.Location);
         }
 
         [Fact]
-        public void TestExeEntryPoint()
+        public void EntryPoint()
         {
             MethodInfo entryPoint = typeof(TestExe).GetTypeInfo().Assembly.EntryPoint;
             Assert.NotNull(entryPoint);
             Assert.Equal(42, entryPoint.Invoke(null, null));
         }
 
-        private static Assembly ThisAssembly()
-        {
-            return typeof(AssemblyTests).GetTypeInfo().Assembly;
-        }
+        private static Assembly GetExecutingAssembly() => typeof(AssemblyTests).GetTypeInfo().Assembly;
```

Comment:
All the other files in the main test project which have this method are called GetExecutingAssembly